library/sync-tasks/src/androidTest/java/org/cru/godtools/sync/task/BaseDataSyncTasksTest.java

```diff
@@ -56,6 +56,7 @@ private Language language(final Long id, final Locale locale, final boolean adde
         lang.setId(id);
         lang.setCode(locale);
         lang.setAdded(added);
+        lang.setLanguageName(locale.getDisplayName());
```

Comment:
this is unnecessary here because you aren't doing any tests with the language name